lib/telemetry/telemetry.go

```diff
@@ -0,0 +1,90 @@
+package telemetry
+
+// PLSREVIEW(kit): Should this package be lib/telemetry, or should we define it at the top level alongside agent
+//  and lib?
+
+import (
+	"time"
+
+	"github.com/armon/go-metrics"
+)
+
+type Label struct {
+	Key   string
+	Value string
+}
```

Comment:
I'm guessing this exists because we want to use these same labels for tracing (or other non-metrics observability), is that right? That would be great to include in the godoc for this type, so we don't lose the context.